### PR TITLE
Final release notes for 1.7.0

### PR DESCRIPTION
**Pull request by msimberg**
Fills in remaining release notes (PR and issue lists will be updated separately as those are a living thing...).

Could everyone please have a look and fill in whatever I've missed. Apologies in advance for anything that I have missed. In particular, @hkaiser could you add a note on the collectives changes (and anything else that you think is relevant)? @Jedi18, @srinivasyadav18 could you have a look to see if I've missed some algorithms that you've adapted?

### CODE REVIEW

**Review by aurianer — APPROVED** [review]
Thanks a lot for doing this!

File: docs/sphinx/releases/whats_new_1_7_0.rst [hunk 1]
```diff
@@ -64,6 +67,10 @@ General changes
 - All ``hpx::init`` and ``hpx::start`` overloads now take ``std::function``\ s
   instead of ``hpx::util::function_nonser``. No changes should be required in
   user code to accommodate this change.
+- ``hpx::util::unwrapping`` and other related unwrapping functionality has been
+  moved up into the ``hpx`` namespace. Names in ``hpx::util`` are still usable
+  with a deprecation warning. This functionality can now be accessed through the
+  ``hpx/unwrap.hpp`` and ``hpx/local/unwrap.hpp`` headers.
 - The default tag for APEX has been update from 2.3.1 to 2.4.0. In particular,
   this fixes a bug which could lead to hangs in distributed runs.
 - The dependency on Boost.ASIO has been replaced with the standalone ASIO
```
[the remark this answers is not in the document]
Comment (msimberg) [reply docs/sphinx/releases/whats_new_1_7_0.rst]: Updated.

**Comment by msimberg** [comment]
@Jedi18, @srinivasyadav18 did you have anything to add? @Jedi18 I'll still update the list of adapted algorithms depending on how many your PRs we merge before the release.

**Comment by Jedi18** [comment]
Thanks! No, don't have anything to add.

**Comment by srinivasyadav18** [comment]
Thank you. No, I do not have anything to add.

**Pull request merged**